R/get_about_pages.r

```diff
@@ -172,12 +180,29 @@ extract_about_links <- function(base_url, timeout_thres = 10) {
         
         } else {
         
-        # FALSE (relative path)
+          # FALSE (relative path)
```

Comment:
There's one more relative path case where the path is given from the current page instead of from root. 

See about link #2 vs about link #1 here: http://milandevries.org/p3/abouttest/

Reply:
The new commit should have solved this problem. I also added #1, #2 cases as unit tests.